services/ssp-b/src/views/index.html.ejs

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport"
+        content="width=device-width, initial-scale=1.0" />
+  <title>
+    <%= title
+        %>
+  </title>
+  <link rel="stylesheet"
+        href="/css/index.css" />
+  <link rel="icon"
+        href="/img/spy.svg" />
+</head>
+
+<body>
+  <h1>
+    <%= title
+        %>
+  </h1>
+  <section>
+    <h2>Supply Side Platform</h2>
+    <p>SSP provides 3rd party tag which running auction via Protected Audience API and decide which
+      Ads to display</p>
+    <p>Ads will embed inside &lt;fencedframe&gt; like below.</p>
+    <ins class="ads">
+      <script class="ssp_tag"
+              src="<%= new URL(`https://${SSP_HOST}:${EXTERNAL_PORT}/ad-tag.js`) %>"></script>
```

Comment:
SSP_B_HOST

Reply:
@kevinkiklee

Reply:
This is updated in the implementation PR